static/app/utils/parseApiError.tsx

```diff
@@ -1,7 +1,7 @@
 import type {ResponseMeta} from 'sentry/api';
 
 export default function parseApiError(resp: ResponseMeta): string {
-  const {detail} = resp?.responseJSON || ({} as object);
+  const {detail} = resp?.responseJSON || ({} as Record<string, any>);
```

Comment:
note: we should probably forbid using the `object` type generally with `eslint`. It’s not what most people think it is. It’s this rule:

https://github.com/getsentry/sentry/blob/a968724f3d35c1a3be1daf70eea61978e60b6b25/eslint.config.mjs#L412

Reply:
Agree! I will just remove this for now as we actually dont even need to cast this in this case anymore